lldb/source/Target/Process.cpp

```diff
@@ -6668,6 +6676,18 @@ Status Process::CalculateCoreFileSaveRanges(lldb::SaveCoreStyle core_style,
   return Status(); // Success!
 }
 
+std::vector<ThreadSP>
+Process::CalculateCoreFileThreadList(const SaveCoreOptions &core_options) {
+  std::vector<ThreadSP> thread_list;
+  for (const auto &thread : m_thread_list.Threads()) {
```

Comment:
change `const auto &thread ` to `const ThreadSP &thread_sp` to indicate "thread" is a shared pointer and not use `auto` unless it saves tons of text